runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -69,13 +69,14 @@
 	private final JPanel statsPanel = new JPanel();
 
 	private final ProgressBar progressBar = new ProgressBar();
+	private final JPopupMenu popupMenu = new JPopupMenu();
 
 	private final JLabel expGained = new JLabel();
 	private final JLabel expHour = new JLabel();
 	private final JLabel expLeft = new JLabel();
 	private final JLabel actionsLeft = new JLabel();
 
-	XpInfoBox(XpTrackerPlugin xpTrackerPlugin, Client client, JPanel panel, Skill skill, SkillIconManager iconManager) throws IOException
+	XpInfoBox(XpTrackerPlugin xpTrackerPlugin, Client client, XpInfoBoxPanel panel, Skill skill, SkillIconManager iconManager) throws IOException
```

Comment:
Any reason why this constructor declares `throws IOException`? I can't see anything that could throw one.